deploy/prometheus_rule.yaml

```diff
@@ -8,11 +8,60 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: marketplace.rules
+    - name: marketplace.community_operators.rules
       rules:
-        - alert: community-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="community-operators"}[10m]) > 10
-        - alert: certified-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="certified-operators"}[10m]) > 10
-        - alert: redhat-operators-alert
-          expr: increase(app_registry_request_total{code!="200",opsrc="redhat-operators"}[10m]) > 10
+        - record: app_registry:community_operators:1xx_response
+          expr: sum(app_registry_request_total{code=~"1..",opsrc="community-operators"})
+        - record: app_registry:community_operators:2xx_response
+          expr: sum(app_registry_request_total{code=~"2..",opsrc="community-operators"})
+        - record: app_registry:community_operators:3xx_response
+          expr: sum(app_registry_request_total{code=~"3..",opsrc="community-operators"})
+        - record: app_registry:community_operators:4xx_response
+          expr: sum(app_registry_request_total{code=~"4..",opsrc="community-operators"})
+        - record: app_registry:community_operators:5xx_response
+          expr: sum(app_registry_request_total{code=~"5..",opsrc="community-operators"})
+        - alert: CommunityOperatorConnectionErrors
+          annotations:
+            message: Unable to connect to the default OperatorSource community-operators AppRegistry for {{ $value }}% of requests.
+          expr: increase(app_registry_request_total{code!~"2..",opsrc="community-operators"}[5m]) / increase(app_registry_request_total{opsrc="community-operators"}[5m]) > 0.5
```

Comment:
Increase is a rather unusual use for this, `rate` is typically used for the purpose that we use it here for.

Reply:
@brancz Addressed in the latest version of this PR, but would you mind sharing the reasoning?